internal/schema-registry/command_dek_delete.go

```diff
@@ -0,0 +1,93 @@
+package schemaregistry
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+	"github.com/confluentinc/cli/v3/pkg/deletion"
+	"github.com/confluentinc/cli/v3/pkg/output"
+	"github.com/confluentinc/cli/v3/pkg/resource"
+)
+
+func (c *command) newDekDeleteCommand(cfg *config.Config) *cobra.Command {
```

Comment:
Support multiple deletion?

Reply:
A pair of name and subject will just specify one DEK, so we can't do it like `delete name-1 name-2`

Reply:
A DEK is identified by a KEK name and a subject. A version can also be passed. When getting DEKs, if no version is passed, then version 1 is returned. When deleting DEKs, if no version is passed, then all versions are deleted.